scripts/track-dependency-status.sh

```diff
@@ -1,58 +1,97 @@
-# Get total count of python dependencies and outdated python dependencies
-
-# skip 2 header lines (produced by pip list)
-total_python_deps="$(pip list | tail -n +3 | wc -l)"
-# skip 1 header line (produced by pip-dep-debt.py)
-outdated_python_deps_list="$(pip list --format json --outdated | ./scripts/pip-dep-debt.py | tail -n +2)"
-outdated_python_deps=$(echo "${outdated_python_deps_list}" | wc -l)
-multi_major_outdated_python_deps=$(echo "${outdated_python_deps_list}" | grep '^[^01]' | wc -l)
```

Comment:
I tracked down differences to metrics this PR is giving compared to previously, and it came down to a bug in this line. It was excluding '13.0.0' from muliti-major, because it starts with `1` 🤦

Reply:
I had a commit that fixed that, and then I just overhauled the entire thing and rebased it out ;)